ha/fault_tolerance/cluster_stop_monitor.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+"""
+Handler for handling cluster stop events received from CSM
+"""
+
+import json
+import ast
+
+from cortx.utils.conf_store import Conf
+from cortx.utils.log import Log
+from ha.util.message_bus import MessageBus, CONSUMER_STATUS, MessageBusConsumer
+
+from ha import const
+from ha.k8s_setup.const import _DELIM
+
+class ClusterStopMonitor:
+
+    def __init__(self):
+        """Init method"""
+        self._consumer = self._get_consumer()
+
+    def start(self):
+        """
+        Start to listen messages.
+        """
+        self._consumer.start()
```

Comment:
I think there is a check here for self._consumer. whether its None or not

Reply:
This was added in fault_monitor [ The derived class was used for getting consumerthere]

Reply:
Got it. Thanks!